README.md

```diff
@@ -84,114 +53,100 @@ To remove a dependency from your project, you can use the `poetry remove` comman
 poetry remove {package_name}
 ```
 
-## Creating and Using Virtual Environments with Poetry
+## Create and Use Virtual Environments with Poetry
 
-You can create a virtual environment for the project by navigating to the project's root directory and run:
+You can create a virtual environment for the project by navigating to the project's root directory and running the following command:
 
 ```bash
 poetry install
 ```
 
 This will create a new virtual environment and install the project's dependencies.
 
-### Activating the Virtual Environment
+### Activate the Virtual Environment
 
-To activate the virtual environment, use `poetry shell`. This will start a new shell session with the virtual
-environment activated:
+To activate the virtual environment, use `poetry shell`. This will start a new shell session with the virtual environment activated:
 
 ```bash
 poetry shell
 ```
 
-You can now run Python and any installed packages in this shell, and they will use the virtual environment. To exit the
-virtual environment, simply `exit` command.
+You can now run Python and any installed packages in this shell. They will use the virtual environment. 
+To exit the virtual environment, simply use the `deactivate` command.
 
-Poetry automatically uses the virtual environment when you run commands like `poetry run`.
+Poetry automatically uses the virtual environment when you run, for example, the `poetry run` command.
 
-Remember to run these commands in the root directory of your project where the `pyproject.toml` file is located.
+Remember to run these commands in the root directory of your project, where the `pyproject.toml` file is located.
 
-Pycharm users can also use the virtual environment created by Poetry. To do this follow
-the [guides](https://www.jetbrains.com/help/pycharm/poetry.html).
+If you are a PyCharm user and want to use the virtual environment created by Poetry, follow the [configuration guides](https://www.jetbrains.com/help/pycharm/poetry.html).
 
-## Using Poe the Poet as Task Runner
+## Use Poe the Poet as Task Runner
 
 [Poe the Poet](https://poethepoet.natn.io/index.html) is a task runner that simplifies running common tasks in a Python project.
 
-To have the command availalbe as `poetry poe <command>` as seen in the following examples, you need to install Poe as a plugin to poetry:
+To have the command available as `poetry poe <command>` as seen in the following examples, you need to install poe as a plugin to Poetry:
 
 ```bash
 poetry self add 'poethepoet[poetry_plugin]'
 ```
 
-If this plugin is not installed you may have to run Poe as a script within the Poetry environment using `poetry run poe <command>`.
-
-## Usage
-
-> Explain how to use the project. You can create multiple subsections (H3). Include the instructions or provide links to
-> the related documentation.
+If this plugin is not installed, you may have to run poe as a script within the Poetry environment using `poetry run poe <command>`.
```

Comment:
"may" - is this uncertain? Is there a chance that it works without the plugin, and even without running it as a script?

Reply:
@grischperl Can you help with this question?

Reply:
"may" is wrong. If you don't have the plugin installed you have to use `poetry run poe <command>`. 

You can only use `poetry poe <command>` with the plugin installed.